TTS/config/shared_configs.py

```diff
@@ -226,7 +226,7 @@ class BaseTrainingConfig(Coqpit):
     run_description: str = ""
     # training params
     epochs: int = 10000
-    batch_size: int = MISSING
+    batch_size: int = None
```

Comment:
Because for the training of the speaker encoder we do not use batch size, instead the number of speakers in the batch is used. I had to set a value so as not to make an error. Do you suggest adding a value instead of None?

Reply:
No it's fine.